Sources/PackageSets/Model/Package.swift

```diff
@@ -0,0 +1,136 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import struct Foundation.URL
+
+import PackageModel
+import SourceControl
+
+extension PackageSetsModel {
+    /// Package metadata
+    public struct Package {
+        /// Package reference
+        public let reference: PackageReference
+
+        /// Package's repository address
+        public let repository: RepositorySpecifier
+
+        /// Package description
+        public let description: String?
+
+        /// Published versions of the package
+        public let versions: [Version]
+
+        /// The latest published version of the package
+        public let latestVersion: Version?
+
+        /// Number of watchers
+        public let watchersCount: Int?
+
+        /// URL of the package's README
+        public let readmeURL: URL?
+
+        /// Package authors
+        public let authors: [Author]?
+
+        /// Initializes a `PackageMetadata`
+        init(
+            repository: RepositorySpecifier,
+            description: String?,
+            versions: [Version],
+            latestVersion: Version?,
+            watchersCount: Int?,
+            readmeURL: URL?,
+            authors: [Author]?
+        ) {
+            self.reference = .init(repository: repository)
+            self.repository = repository
+            self.description = description
+            self.versions = versions
+            self.latestVersion = latestVersion
+            self.watchersCount = watchersCount
+            self.readmeURL = readmeURL
+            self.authors = authors
+        }
+    }
+}
+
+extension PackageSetsModel.Package {
+    /// A representation of package version
+    public struct Version {
```

Comment:
Should this also include the package manifest tools version (what's specified in the top-line comment)?

Reply:
added cc @yim-lee